src/CSharpFunctionalExtensions.FluentAssertions/MaybeExtensions.cs

```diff
@@ -14,6 +14,23 @@ public class MaybeAssertions<T> : ReferenceTypeAssertions<Maybe<T>, MaybeAsserti
     public MaybeAssertions(Maybe<T> instance) : base(instance) { }
 
     protected override string Identifier => "Maybe{T}";
+    
+    /// <summary>
+    /// Asserts that the current <see cref="Maybe{T}"/> has some value.
+    /// </summary>
+    /// <param name="because"></param>
+    /// <param name="becauseArgs"></param>
+    /// <returns></returns>
+    public AndConstraint<MaybeAssertions<T>> HaveSomeValue(string because = "", params object[] becauseArgs)
+    {
+        Execute.Assertion
+            .BecauseOf(because, becauseArgs)
+            .Given(() => Subject)
+            .ForCondition(v => v.HasValue)
+            .FailWith("Expected a value{reason}");
```

Comment:
```suggestion
            .FailWith("Expected a value {reason}");
```

Small whitespace formatting suggestion

Reply:
Ok.